### ISSUE TITLE
not compatible with Laravel 5.2

### ISSUE THREAD

**Issue by salseeg** [benrowe/laravel-filesystem-url#1]
```
PHP Fatal error:  Call to undefined method Illuminate\Foundation\Application::bindShared() in vendor/benrowe/laravel-filesystem-url/src/ServiceProvider.php on line 47
PHP Stack trace:
PHP   1. {main}() artisan:0
PHP   2. Illuminate\Foundation\Console\Kernel->handle() artisan:36
PHP   3. Illuminate\Foundation\Console\Kernel->bootstrap() vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:105
PHP   4. Illuminate\Foundation\Application->bootstrapWith() vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:219
PHP   5. Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap() vendor/laravel/framework/src/Illuminate/Foundation/Application.php:203
PHP   6. Illuminate\Foundation\Application->registerConfiguredProviders() vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/RegisterProviders.php:17
PHP   7. Illuminate\Foundation\ProviderRepository->load() vendor/laravel/framework/src/Illuminate/Foundation/Application.php:530
PHP   8. Illuminate\Foundation\Application->register() vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:74
PHP   9. Benrowe\Laravel\Url\ServiceProvider->register() vendor/laravel/framework/src/Illuminate/Foundation/Application.php:554


  [Symfony\Component\Debug\Exception\FatalErrorException]                   
  Call to undefined method Illuminate\Foundation\Application::bindShared()  
```

**Comment by benrowe** [comment]
Thanks. I've updated the ServiceProvider to support 5.2 & released a new version (v1.0.3)

**Comment by salseeg** [comment]
tnx a lot
